src/darby.js

```diff
@@ -58,6 +66,7 @@ const HELP_ME_REGEX = /(?:^|\s)+(?:encourage|help)\sme/i;
 const HELP_OTHER_REGEX = /(?:^|\s)+(?:encourage|help)\s<@(.*?)>/i;
 const START_ODDS_REGEX = /^\$odds\s+<@(.*?)>\s*(.*)$/i;
 const SET_ODDS_REGEX = /^\$odds\s*([0-9]*)$/i;
+const INT_CHECK_REGEX = /(^[0-9]*$)/i;
```

Comment:
Regex will match everything, should use `+` (one or more) instead of `*` (any). Also, no need for capturing parens (only capturing one thing).